.env.local

```diff
@@ -1,6 +1,6 @@
 NEXT_PUBLIC_ENV=staging
 NEXT_PUBLIC_SERLO_DOMAIN_PRODUCTION=serlo.org
 NEXT_PUBLIC_SERLO_DOMAIN_STAGING=serlo-staging.dev
-HYDRA_CLIENT_ID_STAGING=frontend.serlo.org
-HYDRA_CLIENT_SECRET_STAGING=frontend.serlo.org
-HYDRA_HOST_STAGING=http://localhost:4444
+HYDRA_CLIENT_ID_STAGING=frontend
+HYDRA_CLIENT_SECRET_STAGING=YjgwZThhZDk0MTYxMTA4ODY1NzYxZmJk
+HYDRA_HOST_STAGING=https://hydra.${NEXT_PUBLIC_SERLO_DOMAIN_STAGING}
```

Comment:
Yes, this contains the secret for the staging environment. Obviously, the production environment has a different secret ;)